src/sentry/api/endpoints/organization_details.py

```diff
@@ -570,28 +580,254 @@ def post_org_pending_deletion(
+        required=False,
+    )
+    name = serializers.CharField(
+        max_length=64, help_text="The new name for the organization.", required=False
+    )
+    isEarlyAdopter = serializers.BooleanField(
+        help_text="Specify `true` to opt-in to new features before they're released to the public.",
+        required=False,
+    )
+    aiSuggestedSolution = serializers.BooleanField(
+        help_text="Specify `true` to opt-in to [AI Suggested Solution](/product/issues/issue-details/ai-suggested-solution/) to get AI help on how to solve an issue.",
+        required=False,
+    )
+    codecovAccess = serializers.BooleanField(
+        help_text="Specify `true` to enable Code Coverage Insights. This feature is only available for organizations on the Team plan and above. Learn more about Codecov [here](/product/codecov/).",
+        required=False,
+    )
+
+    # membership
+    defaultRole = serializers.ChoiceField(
+        choices=roles.get_choices(),
+        help_text="The default role new members will receive.",
+        required=False,
+    )
+    openMembership = serializers.BooleanField(
+        help_text="Specify `true` to allow organization members to freely join any team.",
+        required=False,
+    )
+    eventsMemberAdmin = serializers.BooleanField(
+        help_text="Specify `true` to allow members to delete events (including the delete & discard action) by granting them the `event:admin` scope.",
+        required=False,
+    )
+    alertsMemberWrite = serializers.BooleanField(
+        help_text="Specify `true` to allow members to create, edit, and delete alert rules by granting them the `alerts:write` scope.",
+        required=False,
+    )
+    attachmentsRole = serializers.ChoiceField(
+        choices=roles.get_choices(),
+        help_text="The role required to download event attachments, such as native crash reports or log files.",
+        required=False,
+    )
+    debugFilesRole = serializers.ChoiceField(
+        choices=roles.get_choices(),
+        help_text="The role required tto download debug information files, proguard mappings and source maps.",
+        required=False,
+    )
+
+    # avatar
+    avatarType = serializers.ChoiceField(
+        choices=(("letter_avatar", "Use initials"), ("upload", "Upload an image")),
+        help_text="The type of display picture for the organization",
+        required=False,
+    )
+    avatar = serializers.CharField(
+        help_text="The image to upload as the organization avatar, in base64. Required if `avatarType` is `upload`.",
+        required=False,
+    )
+
+    # security & privacy
+    require2FA = serializers.BooleanField(
+        help_text="Specify `true` to require and enforce two-factor authentication for all members.",
+        required=False,
+    )
+    allowSharedIssues = serializers.BooleanField(
+        help_text="Specify `true` to allow sharing of limited details on issues to anonymous users.",
```

Comment:
What are anonymous users? This is the first time I've seen this mentioned. Is it users who aren't signed in?

Reply:
yup exactly users who aren't signed in! i used the exact language used in the UI. it's for whether you can share public links of issues or not.